packages/types/src/cart/common.ts

```diff
@@ -336,6 +336,16 @@ export interface CartLineItemDTO {
    * @expandable
    */
   adjustments?: LineItemAdjustmentLineDTO[]
+  /**
+   * The associated cart.
+   *
+   * @expandable
+   */
+  cart?: CartDTO
+  /**
+   * The ID of the associated cart.
+   */
+  cart_id: string
```

Comment:
question: should it be optional to?

Reply:
Why would you like it to be optional?

Reply:
because the cart is optional, so it means that the cart id too no?

Reply:
I might have misunderstood the way we define expandable vs non-expandable properties - my bad. Line items will always have a cart.